backend/src/main/java/com/google/blackswan/servlets/CronServlet.java

```diff
@@ -38,19 +42,52 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // TODO: Update or clean up log message. 
     log.info("Cron job ran.");
-    
-    // TODO: Logic for cron job to run blackswan mock. (#13)
-    storeAlertInDatastore();
-    
+
+    // Simple logic for cron job, since we only have one set of alerts for now. 
+    clearCurrentAlertsInDatastore(); // Need to clear alerts as we only have one set of alerts now. 
+    storeAlertsInDatastoreSimple();
     response.setStatus(HttpServletResponse.SC_ACCEPTED); 
   }
 
-  /** Temporary method for storing dummy alert into datastore. */
-  private void storeAlertInDatastore() {
-    AlertGenerator testAlertGenerator = new DummyAlertGenerator(new DummyAnomalyGenerator());
-    List<Alert> alerts = testAlertGenerator.getAlerts();
+  private void clearCurrentAlertsInDatastore() {
+    Query query = new Query(Alert.ALERT_ENTITY_KIND);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    // Iterate through the entities to delete all comment objects.
+    for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      Key alertEntityKey = KeyFactory.createKey(Alert.ALERT_ENTITY_KIND, id);
+      datastore.delete(alertEntityKey);
+    }
+
+  }
+
+  /** Storing alerts from simpleGenerator into the datastore. */
+  private void storeAlertsInDatastoreSimple() {
+    AlertGenerator simpleGenerator = new SimpleAlertGenerator(new SimpleAnomalyGenerator());
+    simpleGenerator.getAlerts().forEach(alert -> {
+      DatastoreServiceFactory.getDatastoreService().put(alert.toEntity());
+    });
+  }
+
+  /** 
+   * Sample fetch alerts from datastore and creating alert objects from it. 
+   * Used during demo for MVP presentation. TODO: delete or modify. 
+   */
+  private void fetchAlertsFromDatastore() {
+    Query query = new Query(Alert.ALERT_ENTITY_KIND);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+    List<Alert> alertList = new ArrayList<>();
+
+    for (Entity entity : results.asIterable()) {
+      alertList.add(Alert.createAlertFromEntity(entity));
+    }
 
-    DatastoreServiceFactory.getDatastoreService().put(alerts.get(0).toEntity());
+    // Print out to see if alerts are correctly converted.
+    // Used primarily during backend demo. 
+    alertList.forEach(alert -> System.out.println(alert));
```

Comment:
seems we're not using alertsList other than for printing out the alert.

could we just not store the alert list and do the print ln, or are you storing the alertList since you're planning to do other things with the alertList later on?
```
for (Entity entity : results.asIterable()) {
    System.out.println(Alert.createAlertFromEntity(entityt));
}
```

Reply:
Yep, that makes sense. The only reason that I made it into an `alertList` was because i was testing conversion of the whole `alertList` to JSON with GSON, but I removed the GSON code and forgot to get rid of `alertList`.

I will remove this function as a whole since the mock does not need to fetch alerts from the datastore, this was just in place for demo purposes.